data/en/lms_2.json

```diff
@@ -21,11 +21,31 @@
             "validator": "string"
         },
         {
-            "name": "ru_name",
+            "name": "country",
             "validator": "string"
         },
         {
-            "name": "country",
+            "name": "address_line1",
+            "validator": "optional_string"
+        },
+        {
+            "name": "address_line2",
+            "validator": "optional_string"
```

Comment:
Isn't `optional` defined in the schema as a new property (https://github.com/ONSdigital/eq-schema-validator/blob/master/schemas/questionnaire_v1.json#L78), so would look like
```
{
            "name": "address_line2",
            "validator": "string"
			"optional": true
}
```
in a survey schema?

Reply:
Yeah, so this was what we had originally (I added the `"optional": true`) but after a couple of discussions we went with creating `"validator": "optional_string"`

Reply:
If you can remember, Why did you go with this instead of the first version?

Reply:
Instead of adding that additional field, it was added to the existing list of types, there's validation that's done in metadata_parser.py(~L44) which does validation on each of these types, so it made sense to add it to there